docs/other-api/adapter.md

```diff
@@ -16,7 +16,7 @@ Idiomatic Remix apps can generally be deployed anywhere because Remix adapt's th
 - `@remix-run/netlify`
 - `@remix-run/vercel`
 
-These adapters are imported into your server's entry and is not used inside of your Remix app itself.
+These adapters are imported into your server's entry and are not used inside of your Remix app itself.
```

Comment:
I'm assuming it should be like
"These adapters ... are"
or
"An adapter ... is"